bundlegen/src/main/java/com/citrix/microapps/bundlegen/pojo/ModelTranslation.java

```diff
@@ -0,0 +1,24 @@
+package com.citrix.microapps.bundlegen.pojo;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+public class ModelTranslation {
+
+    private Map<String, Map<String, String>> appTranslations;
+
+    @JsonCreator
+    public ModelTranslation(Map<String, Map<String, String>> appTranslations) {
+        this.appTranslations = appTranslations;
+    }
+
+    public Map<String, Map<String, String>> getAppTranslations() {
+        return appTranslations;
+    }
+
+    public void setAppTranslations(Map<String, Map<String, String>> translations) {
```

Comment:
The setter should not be needed with `@JsonCreator`, `appTranslations` can be final.

Reply:
Removed setter.

Reply:
Done.